zhmccli/_cmd_partition.py

```diff
@@ -439,6 +441,42 @@ def partition_console(cmd_ctx, cpc, partition, **options):
                                                       options))
 
 
+@partition_group.command('mountiso', options_metavar=COMMAND_OPTIONS_METAVAR)
+@click.argument('CPC', type=str, metavar='CPC')
+@click.argument('PARTITION', type=str, metavar='PARTITION')
+@click.option('--imagefile', type=str, required=True,
+              help='The path and name of the ISO image.')
+@click.option('--imageinsfile', type=str, required=True,
+              help='The path and name of the ins file.')
+@click.pass_obj
+def partition_mount_iso(cmd_ctx, cpc, partition, **options):
+    """
+    Mount an ISO image to a partition.
+
+    In addition to the command-specific options shown in this help text, the
+    general options (see 'zhmc --help') can also be specified right after the
+    'zhmc' command name.
+    """
+    cmd_ctx.execute_cmd(lambda: cmd_partition_mount_iso(cmd_ctx, cpc,
+                                                        partition, options))
+
+
+@partition_group.command('unmountiso', options_metavar=COMMAND_OPTIONS_METAVAR)
+@click.argument('CPC', type=str, metavar='CPC')
+@click.argument('PARTITION', type=str, metavar='PARTITION')
+@click.pass_obj
+def partition_unmount_iso(cmd_ctx, cpc, partition):
+    """
+    Unmount an ISO image from a partition.
```

Comment:
Add: "The boot options of the partition will be set to not boot from anything."

Reply:
Withdrawing this comment, because we now use specific options to set/unset the `boot-device` property.